java/dev/checkstyle/checkstyle.xml

```diff
@@ -247,7 +247,7 @@
             <property name="ignoreInlineTags" value="false"/>
         </module>
         <module name="EmptyCatchBlock">
-            <property name="exceptionVariableName" value="expected"/>
+            <property name="exceptionVariableName" value="expected|ignore"/>
```

Comment:
Is it more consistent to name it "expected" and "ignored"?  Or maybe remove this rule? Personally, doesn't seem too useful to me...

Reply:
I think it's somewhat useful to convey that there is an empty catch on purpose. We actually had an instance or two of using `ignore` plus both were given as an example in the docs http://checkstyle.sourceforge.net/config_blocks.html#EmptyCatchBlock.  Either sound fine to me, which is why I added it. Are you suggesting to only use `except`?  There are not too many instances, so not a big deal.

Reply:
I see. If "ignore" is in the example. I am fine with it. I just found it bit inconsistent to have `expected|ignore` instead of `expected|ignored` but it's not a bit deal.